SonarQube.MSBuild.Tasks/Resources.resx

```diff
@@ -117,9 +117,6 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="ERROR_InvalidProjectLanguage" xml:space="preserve">
-    <value>ProjectLanguage must be set to "C#" or "VB", not: {0}</value>
-  </data>
```

Comment:
The task will no longer error on unrecognised languages: it will create a ProjectInfo.xml file with the ProjectLanguage set to whatever the $(Language) property is set to (including null).